{
  "execution_info": {
    "timestamp": "2025-05-29 21:43:08",
    "total_duration": 2.580557107925415,
    "environment": "local"
  },
  "summary": {
    "total_categories": 1,
    "passed": 0,
    "failed": 1,
    "success_rate": 0.0
  },
  "category_results": {
    "unit": {
      "success": false,
      "duration": 2.579131603240967,
      "error": "Traceback (most recent call last):\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/bin/pytest\", line 8, in <module>\n    sys.exit(console_main())\n             ^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 201, in console_main\n    code = main()\n           ^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 156, in main\n    config = _prepareconfig(args, plugins)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 341, in _prepareconfig\n    config = pluginmanager.hook.pytest_cmdline_parse(\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/helpconfig.py\", line 105, in pytest_cmdline_parse\n    config = yield\n             ^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 1140, in pytest_cmdline_parse\n    self.parse(args)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 1494, in parse\n    self._preparse(args, addopts=addopts)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 1398, in _preparse\n    self.hook.pytest_load_initial_conftests(\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_hooks.py\", line 512, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    raise exception\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/warnings.py\", line 151, in pytest_load_initial_conftests\n    return (yield)\n            ^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/capture.py\", line 172, in pytest_load_initial_conftests\n    yield\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 1222, in pytest_load_initial_conftests\n    self.pluginmanager._set_initial_conftests(\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 581, in _set_initial_conftests\n    self._try_load_conftest(\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 619, in _try_load_conftest\n    self._loadconftestmodules(\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 659, in _loadconftestmodules\n    mod = self._importconftest(\n          ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 735, in _importconftest\n    self.consider_conftest(mod, registration_name=conftestpath_plugin_name)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 816, in consider_conftest\n    self.register(conftestmodule, name=registration_name)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 512, in register\n    self.consider_module(plugin)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 824, in consider_module\n    self._import_plugin_specs(getattr(mod, \"pytest_plugins\", []))\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 831, in _import_plugin_specs\n    self.import_plugin(import_spec)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 868, in import_plugin\n    self.register(mod, modname)\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/config/__init__.py\", line 501, in register\n    plugin_name = super().register(plugin, name)\n                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pluggy/_manager.py\", line 146, in register\n    raise ValueError(\nValueError: Plugin already registered under a different name: pytest_timeout=<module 'pytest_timeout' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_timeout.py'>\n{'135746848593760': <_pytest.config.PytestPluginManager object at 0x7b7606b09760>, 'pytestconfig': <_pytest.config.Config object at 0x7b7606368bf0>, 'mark': <module '_pytest.mark' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/mark/__init__.py'>, 'main': <module '_pytest.main' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/main.py'>, 'runner': <module '_pytest.runner' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/runner.py'>, 'fixtures': <module '_pytest.fixtures' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/fixtures.py'>, 'helpconfig': <module '_pytest.helpconfig' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/helpconfig.py'>, 'python': <module '_pytest.python' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/python.py'>, 'terminal': <module '_pytest.terminal' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/terminal.py'>, 'debugging': <module '_pytest.debugging' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/debugging.py'>, 'unittest': <module '_pytest.unittest' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/unittest.py'>, 'capture': <module '_pytest.capture' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/capture.py'>, 'skipping': <module '_pytest.skipping' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/skipping.py'>, 'legacypath': <module '_pytest.legacypath' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/legacypath.py'>, 'tmpdir': <module '_pytest.tmpdir' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/tmpdir.py'>, 'monkeypatch': <module '_pytest.monkeypatch' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/monkeypatch.py'>, 'recwarn': <module '_pytest.recwarn' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/recwarn.py'>, 'pastebin': <module '_pytest.pastebin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/pastebin.py'>, 'assertion': <module '_pytest.assertion' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/assertion/__init__.py'>, 'junitxml': <module '_pytest.junitxml' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/junitxml.py'>, 'doctest': <module '_pytest.doctest' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/doctest.py'>, 'cacheprovider': <module '_pytest.cacheprovider' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/cacheprovider.py'>, 'freeze_support': <module '_pytest.freeze_support' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/freeze_support.py'>, 'setuponly': <module '_pytest.setuponly' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/setuponly.py'>, 'setupplan': <module '_pytest.setupplan' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/setupplan.py'>, 'stepwise': <module '_pytest.stepwise' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/stepwise.py'>, 'warnings': <module '_pytest.warnings' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/warnings.py'>, 'logging': <module '_pytest.logging' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/logging.py'>, 'reports': <module '_pytest.reports' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/reports.py'>, 'python_path': <module '_pytest.python_path' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/python_path.py'>, 'unraisableexception': <module '_pytest.unraisableexception' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/unraisableexception.py'>, 'threadexception': <module '_pytest.threadexception' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/threadexception.py'>, 'faulthandler': <module '_pytest.faulthandler' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/_pytest/faulthandler.py'>, 'pytest_mock': <module 'pytest_mock' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_mock/__init__.py'>, 'pytest_cov': <module 'pytest_cov.plugin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_cov/plugin.py'>, 'benchmark': <module 'pytest_benchmark.plugin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_benchmark/plugin.py'>, 'xdist': <module 'xdist.plugin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/xdist/plugin.py'>, 'xdist.looponfail': <module 'xdist.looponfail' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/xdist/looponfail.py'>, 'timeout': <module 'pytest_timeout' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_timeout.py'>, 'faker': <module 'faker.contrib.pytest.plugin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/faker/contrib/pytest/plugin.py'>, 'asyncio': <module 'pytest_asyncio.plugin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_asyncio/plugin.py'>, 'anyio': <module 'anyio.pytest_plugin' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/anyio/pytest_plugin.py'>, 'capturemanager': <CaptureManager _method='fd' _global_capturing=<MultiCapture out=<FDCapture 1 oldfd=5 _state='started' tmpfile=<_io.TextIOWrapper name=\"<_io.FileIO name=6 mode='rb+' closefd=True>\" mode='r+' encoding='utf-8'>> err=<FDCapture 2 oldfd=7 _state='started' tmpfile=<_io.TextIOWrapper name=\"<_io.FileIO name=8 mode='rb+' closefd=True>\" mode='r+' encoding='utf-8'>> in_=<FDCapture 0 oldfd=3 _state='started' tmpfile=<_io.TextIOWrapper name='/dev/null' mode='r' encoding='utf-8'>> _state='started' _in_suspended=False> _capture_fixture=None>, '_cov': <pytest_cov.plugin.CovPlugin object at 0x7b7605004620>, '/home/user/Escritorio/hw/backend/tests/conftest.py': <module 'tests.conftest' from '/home/user/Escritorio/hw/backend/tests/conftest.py'>, 'pytest_asyncio': <module 'pytest_asyncio' from '/home/user/.cache/pypoetry/virtualenvs/licitacao-backend-dqaKrIxa-py3.12/lib/python3.12/site-packages/pytest_asyncio/__init__.py'>}\n"
    }
  },
  "quality_gates": {
    "api_contracts": {
      "passed": false,
      "criteria_met": false,
      "details": "Category not executed"
    },
    "security": {
      "passed": false,
      "criteria_met": false,
      "details": "Category not executed"
    },
    "performance": {
      "passed": false,
      "criteria_met": false,
      "details": "Category not executed"
    },
    "chaos": {
      "passed": false,
      "criteria_met": false,
      "details": "Category not executed"
    },
    "stress": {
      "passed": false,
      "criteria_met": false,
      "details": "Category not executed"
    }
  },
  "recommendations": [
    "Investigate and fix 1 failed test categories"
  ]
}